Some light reorganisation of stubtest allowlists (#10260)
python/typeshed: tests/stubtest_allowlists/py3_common.txt

multiprocessing.managers.DictProxy.popitem
multiprocessing.managers.DictProxy.values

multiprocessing.(dummy|managers).Namespace.__[gs]etattr__  # Any field can be set on Namespace

> # alias for a class defined elsewhere, mypy infers the variable has type `(*args) -> ForkingPickler` but stubtest infers the runtime type as <class ForkingPickler>

Reviewer: line-length nit

```suggestion
# alias for a class defined elsewhere,
# mypy infers the variable has type `(*args) -> ForkingPickler`
# but stubtest infers the runtime type as <class ForkingPickler>
```

sys.ps2
sys.tracebacklimit

sys.implementation  # Actually SimpleNamespace but then you wouldn't have convenient attributes

> # A factory function that returns 'most efficient lock'. Marking it as a function will make it harder for users to mark the Lock type.

Reviewer: ```suggestion
# A factory function that returns 'most efficient lock'.
# Marking it as a function will make it impossible for users to use the Lock type as an annotation.
```

xml.etree.cElementTree.Element.__iter__
typing.IO.__iter__  # See https://github.com/python/typeshed/commit/97bc450acd60c1bcdafef3ce8fbe3b95a9c0cac3

# LC_MESSAGES is sometimes present in __all__, sometimes not,
# so stubtest will sometimes complain about exported names being different at runtime to the exported names in the stub
(locale.__all__)?

> (sys.get_int_max_str_digits)?  # Added in a patch release, backported to all security branches, but has yet to find its way to all GitHub Actions images
> (sys.set_int_max_str_digits)?  # Added in a patch release, backported to all security branches, but has yet to find its way to all GitHub Actions images

Reviewer: ```suggestion
# Added in a patch release, backported to all security branches,
# but have yet to find their way to all GitHub Actions images
(sys.get_int_max_str_digits)?
(sys.set_int_max_str_digits)? 
```
